zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -285,6 +301,29 @@ public boolean deleteCollection(HttpServletRequest request, HttpServletResponse
         return true;
     }
 
+    private void deleteCollection(com.github.onsdigital.zebedee.model.Collection c, Session s)
+            throws InternalServerError, NotFoundException, BadRequestException {
```

Comment:
What about unauthorised error?

Reply:
Does it fall under BadRequestException?

Reply:
None of the methods called here will thrown an Unauthorised exception so it doesn't need to be added to the method signature.

Reply:
But when the session is null, it throws an unauthorised exception according to the tests....